### PR TITLE
change vm name for gce

### PR DESCRIPTION
**Pull request by kzvyahin**
{{pytest: cfme/tests/cloud_infra_common/test_genealogy.py::test_vm_genealogy_detected --use-provider="gce_central"}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 14acdf68401316ace80dd8bc54719ad0b2635d66:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/cloud_infra_common/test_genealogy.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def pytest_generate_tests(metafunc):
 @pytest.fixture(scope="function")
 def vm_crud(provider, small_template):
     return VM.factory(
-        'test_genealogy_{}'.format(fauxfactory.gen_alpha(length=8).lower()),
+        'test-genealogy-{}'.format(fauxfactory.gen_alpha(length=8).lower()),
```
Comment (jkrocil) [inline cfme/tests/cloud_infra_common/test_genealogy.py]: Use `https://github.com/ManageIQ/integration_tests/blob/master/utils/generators.py#L6` instead, please.

Comment (jkrocil) [inline cfme/tests/cloud_infra_common/test_genealogy.py]: `random_vm_name(context='genealogy')` will give you `test-genealogy-a9bwq0`.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 2d40bbd859191ce530aa8d0946219614f35b50a4:

`cfme/tests/cloud_infra_common/test_genealogy.py`:
- :red_circle: Line 2:1: **F401** *'fauxfactory' imported but unused*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/cloud_infra_common/test_genealogy.py (hunk 1 above)
Comment (mfalesni) [inline cfme/tests/cloud_infra_common/test_genealogy.py]: and the context= is unnecessary in this case, just prolongs the line :)

**Comment by dajoRH** [comment]
Lint report for commit d3a2170a374d02dc5719183047d9d7d78da5a036:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f4a16d0f6735b47ab5dd0c9fe4a52133c54e10a4:

`cfme/tests/cloud_infra_common/test_genealogy.py`:
- :red_circle: Line 35:23: **F821** *undefined name 'random_vm_name'*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit bda83efe05cda91d7d543dc4fd75f9387c46aab6:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 75a948e57996be707b5fc537bd4c85e57ae2f554:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Pull request merged**